docs/content/best-practices.md

```diff
@@ -27,12 +27,13 @@ That means:
 
 1. **Use `<table border="0" cellpadding="0" cellspacing="0" role="presentation">` when creating new tables.** This negates any unwanted spacing and borders and tells screen readers to skip over the table’s tags and move straight into the content.
 2. **When in doubt, nest another table.** For finer control of your HTML, nest tables when building emails.
-3. **Use padding for spacing in table cells.** Margins aren’t fully supported on tables and container elements.
-4. **Use margin for typography.** Margins *are* fully supported for headlines, paragraphs, and lists.
-5. **Use `align` for layout instead of `float`, `grid`, or `flexbox`.** Floats aren’t supported in Outlook and email clients don’t have good support for modern CSS layout properties in general.
-6. **HTML attributes are still relevant.** Most styling is done using CSS. But because some email clients use antiquated rendering engines, they tend to better understand HTML attributes like `align`, `valign`, `height`, and `width`.
-7. **Define color as `#ffffff` instead of `#fff` or `rgb(1,2,3)`.** Six-digit hex is supported in inline CSS as well as HTML attributes like `bgcolor` that are still supported in email.
-8. **Don’t forget about [preview text](https://stackoverflow.design/email/guidelines/faq#what-is-preview-text?).** We can specify the text that appears beneath subject lines in many email clients. If preview text is not included, this space will be populated by the email’s content.
+3. **Don't rely on CSS inheritance.** Some versions of Windows Outlook reset CSS properties `font-family`, `font-size`, `font-weight`, `line-height`, `color` from parent elements from `<body>` when nesting tables.
```

Comment:
Thanks for your feedback @TedGoas.

Shouldn't it be "from parent elements **to** `<body>`" rather than "from parent elements from `<body>`"?

What do you think about the following version ?
> Some versions of Windows Outlook reset CSS properties **like** `font-family`, `font-size`, `font-weight`, `line-height`, **and** `color` **that should be inherited** from parent elements **to** `<body>` when nesting tables.

(the bold font is used only to highlight my changes)

I'm not a native speaker so maybe your wording is all right.

Also, I would like to share my personal experience: As a beginner I was not really sure whether to apply the font styles to `<table>`, `<tr>`, or `<td>` elements. Only after I found the [janogarcia's guidelines](https://gist.github.com/janogarcia/4977a2346cbc7e52334b#typography) I realized that the font styles should be applied to `<td>` elements and it helped me to clean up my template quite a lot. Before that, I had the styles scattered randomly among those three element types and sometimes even duplicated both in `<table>` and the descendant `<td>` because I thought that it was necessary to work in different quirky email clients.
That is why I mentioned `<td>` specifically in my wording.

Reply:
Yes, these are good edits, I'll get them in there. Your English is very good :)